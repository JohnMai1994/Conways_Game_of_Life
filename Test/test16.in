new 20
init
10 10 10 11 10 12 10 13 10 14
13 10 13 11 13 12 13 13 13 14
-1 -1
print
step
print
step
print
step
print
step
print
steps 5
print

