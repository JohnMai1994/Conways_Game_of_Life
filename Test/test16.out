____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
__________X__X______
__________X__X______
__________X__X______
__________X__X______
__________X__X______
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
_________XXXXXX_____
_________XXXXXX_____
_________XXXXXX_____
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
__________XXXX______
_________X____X_____
________X______X____
_________X____X_____
__________XXXX______
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
___________XX_______
__________XXXX______
_________XXXXXX_____
________XX____XX____
_________XXXXXX_____
__________XXXX______
___________XX_______
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
__________X__X______
_________X____X_____
________X______X____
________X______X____
________X______X____
_________X____X_____
__________X__X______
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
____________________
_______X________X___
_____XXX________XXX_
_______X________X___
____________________
____________________
____________________
____________________
____________________
____________________
